src/Raven.Server/Smuggler/Documents/DatabaseSource.cs

```diff
@@ -499,6 +500,61 @@ public IEnumerable<TimeSeriesItem> GetTimeSeries(List<string> collectionsToExpor
             }
         }
 
+
+        private static unsafe string GetOriginalName(DocumentsOperationContext context, string docId, string lowerName)
+        {
+            try
+            {
+                try
+                {
+                    using (DocumentIdWorker.GetLower(context.Allocator, docId, out var docIdSlice))
+                    {
+                        context.DocumentDatabase.DocumentsStorage.GetTableValueReaderForDocument(context, docIdSlice, throwOnConflict: true, out var tvr);
+                        var doc = new BlittableJsonReaderObject(tvr.Read((int)DocumentsStorage.DocumentsTable.Data, out int size), size, context);
+                        var name = GetOriginalName(lowerName, doc);
+                        if (name != null)
+                            return name;
+                    }
+                }
+                catch (DocumentConflictException)
+                {
+                    // will try to get it from the conflict storage
+                    foreach (var conflict in context.DocumentDatabase.DocumentsStorage.ConflictsStorage.GetConflictsFor(context, lowerName))
+                    {
+                        if (conflict.Doc == null)
+                            continue;
+
+                        var name = GetOriginalName(lowerName, conflict.Doc);
+                        if (name != null)
+                            return name;
+                    }
+                }
+            }
+            catch
+            {
+#if DEBUG
+                throw;
+#endif
+            }
+
+            Debug.Assert(false, "Are you exporting an orphaned time-series?");
+            return lowerName;
```

Comment:
If this is the case, need to actually throw real error here.
Otherwise, in release, we'll do horrible things here.

Reply:
It is only a precaution, but in case it did happened, do we want to fail the export because of this? if this happened in production, we need a way to fix it.
Worst case we will export time-series that doesn't have any document (and with the lowered id name)

Reply:
If this is fine, then don't throw, just log a note about this.
I really don't like debug / release different behaviors. They lead us to miss important issues.